robottelo/ui/subscription.py

```diff
@@ -1,13 +1,21 @@
 """Implements Subscriptions/Manifest handling for the UI"""
 
-from robottelo.helpers import escape_search
 from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
 
 
 class Subscriptions(Base):
```

Comment:
Was experimenting with `is_katello = False` and forgot to return back, thanks for catching :+1: 
It was passing as we aren't defining subscription names, but using existing from imported manifest, which is always <30 chars for our manifest. But of course it's a bad idea to leave it like this.